ml4gw/transforms/snr_rescaler.py

```diff
@@ -12,16 +12,19 @@ class SnrRescaler(FittableTransform):
     def __init__(
         self,
         n_ifos: int,
```

Comment:
I don't know if this comes up anywhere else in the repo, but JIC it does we should land on a standardization between using `n_` and `num_`

Reply:
I can take a look